bedrock/firefox/views.py

```diff
@@ -542,8 +542,11 @@ def new(request):
                 template = 'firefox/new/batm/scene2.html'
             else:
                 template = 'firefox/new/onboarding/scene2.html'
-        elif lang_file_is_active('firefox/new/onboarding', locale):
-            template = 'firefox/new/onboarding/scene2.html'
+        elif lang_file_is_active('firefox/new/onboarding' 'firefox/new/batm', locale):
+            if experience == 'batmprivate':
+                template = 'firefox/new/batm/scene2.html'
+            else:
+                template = 'firefox/new/onboarding/scene2.html'
```

Comment:
Yeah, this was the bit I was least confident about. It seems to get the expected result but just felt wrong and I wasn't sure what I was missing... now I realize I was missing the `and` bit, rather than nesting a second if. I'll update to your version which is much more sound. Thanks!